public/scripts/week-calendar.js

```diff
@@ -985,44 +1063,59 @@ function checkReminderValidations(form) {
     "reminder_repeatingIncrementBy_error"
   );
 
+  let reminder_priority_error = document.getElementById(
+    "reminder_priority_error"
+  );
   let reminder_endDateTime_error = document.getElementById(
     "reminder_endDateTime_error"
   );
-  form.textBody.setCustomValidity("");
-  form.tag.setCustomValidity("");
-  form.title.setCustomValidity("");
-  form.dateAddedTo.setCustomValidity("");
+
```

Comment:
you need to clear the inner text for each error div and setCustomValidity("") for each form element before doing validations

Reply:
since you are setting everything to empty here.. you dont need the else blocks to set the errorDiv Innertext to "" 
so the else blocks below can be removed

Reply:
or you can add it back to each else block to set customvalidity as "" 
i would prefer the first option i gave